build/sources.ini

```diff
@@ -44,3 +44,7 @@ grammar = https://raw.githubusercontent.com/tree-sitter/tree-sitter-toml/v0.5.1/
 [json]
 nodes = https://raw.githubusercontent.com/tree-sitter/tree-sitter-json/v0.20.2/src/node-types.json
 grammar = https://raw.githubusercontent.com/tree-sitter/tree-sitter-json/v0.20.2/src/grammar.json
+
+[ocaml]
```

Comment:
Shouldn't you also add the other 2 grammars here? (for `.mli` and `.mly`)

Reply:
I think so though I'm not sure how to reconcile having multiple grammars for OCaml when none of the existing languages do. Using a forked `ocaml-ts-mode` there is a major mode for each file type so it probably should work out fine. 

I'm still reading the combobulate code and seeing what works or doesn't work.